tests/integration/test_analysisd/test_eps/data/configuration_template/configuration_enabled.yaml

```diff
@@ -0,0 +1,49 @@
+- sections:
```

Comment:
You do not disable the logcollector module. This could produce that in certain events analysis analyzing more events that the sent with the simulated agent's script.